runtime/permissions/lib.rs

```diff
@@ -787,10 +789,13 @@ impl<TAllowDesc: AllowDescriptor> UnaryPermissionDesc<TAllowDesc> {
     match self {
       UnaryPermissionDesc::Granted(desc) => AllowOrDenyDescRef::Allow(desc),
       UnaryPermissionDesc::FlagDenied(desc) => {
-        AllowOrDenyDescRef::Deny(desc, false)
+        AllowOrDenyDescRef::Deny(desc, 0)
```

Comment:
Consider using an enum for precedence instead of a number, it's not very descriptive :(